docker-compose.yaml.template

```diff
@@ -18,7 +18,12 @@ services:
   backend:
     image: 127.0.0.1:6000/save-backend:{{project.version}}
     environment:
-      - SPRING_PROFILES_ACTIVE={{profile}}
+      - "SPRING_PROFILES_ACTIVE={{profile}}"
+      - "MYSQL_USER: /run/secrets/db_username"
```

Comment:
Is there any difference in `=` and `:` syntax? If not, it'd be better to keep it consistent